src/commcare_cloud/ansible/deploy_riakcs.yml

```diff
@@ -119,12 +119,6 @@
         name: riak
         drop_conf: "{{ backend_vars.drop_conf }}"
         advanced_conf: "{{ backend_vars.advanced_conf }}"
-        data_dirs:
-          - "{{ riak_data_dir }}"
-          - "{{ riak_ring_dir }}"
-          - "{{ riak_data_root_leveldb }}"
-          - "{{ riak_data_root_bitcask if _riak_backend == 'bitcask' else '' }}"
-          - "{{ riak_data_root_blocks if _riak_backend == 'leveldb' else '' }}"
         when: _force_riak_config | bool
```

Comment:
Is it possible these were not getting passed through just because this conditional was evaluating to false? If you run the original playbook with this as true, does it work? It may not be a bug in the playbook, but in the way commcare-cloud uses it